README.md

```diff
@@ -54,6 +55,8 @@ Multi-schema consumers support multiple message types via handler configs. They
     * `options`, composed by
         * `handlers` – configuration for handling each of the supported message types. See "Multi-schema handler definition" for more details;
         * `messageTypeField` - which field in the message describes the type of a message. This field needs to be defined as `z.literal` in the schema and is used for routing the message to the correct handler;
+        * `messageTimestampField` - which field in the message contain the message creation date. This field needs to be ISO-8601 date string and by default it is `timestamp`;
+        * `maxRetryDuration` - how long (in seconds) the message should be retried due to the `retryLater` result before marking it as consumed (avoid infinite loops). Default is 4 days;
```

Comment:
we mark as consumed and not DLQ it?

Reply:
Yes, it goes to DLQ (only on SNS and AMQP) as any other message that can't be consumed, but only if DLQ is configured, already mentioned as a note on DLQ section 🙌 good catch Igor 🙏